packages/synchro-charts/src/components/sc-webgl-context/sc-webgl-context.css

```diff
@@ -16,9 +16,16 @@ sc-webgl-context {
   top: 0;
 
   /* Prevent the canvas from stretching beyond the window causing infinite scrolling bugs */
-  width: calc(100vw - var(--scroll-bar-size));
-  height: calc(100vh - var(--scroll-bar-size));
```

Comment:
have you tested this in a scenario where there is a scroll bar on the bottom of the screen?

Reply:
Added this back into the `.webgl-context-canvas-viewport` which is what's used when the viewport is the view frame (how it works today). So this bug should still be accounted for